docs/LISTING-README-TEMPLATE.md

```diff
@@ -0,0 +1,98 @@
+# Description
+**Focus on what this software is and the VALUE it brings to users.**
+------
+(replace all text in italics.)
+
+_Write a few paragraphs about *WHAT* the listing is, *WHAT* the software does, and the *VALUE* it brings to users._
+
+_Include specific details on what this 1-click provides, and the value to users. Enumerate the values this 1-click's config gives the user. Enumerate the value this software combination, specifically for stacks containing multiple pieces of software, bring to the user._ 
+
+_Include links for the user to learn more about the software that are not included in the software included section._
+
+_Recommend CPU and Memory minimum requirements and node pools for test/dev and production for X amount of user._
+
+_If your software is Open Source a thank you to all the contributors is a nice touch. Suggest to add something to the effect:_
+Thank you to all the contributors whose hard work make this software valuable for users.
+
+
+# Getting Started
+(Replace `[service]` with the specific service name.)
```

Comment:
Not a blocker, but we may want to use `[app]` instead of `[service]` for consistency.

Reply:
Updated.